.github/assets/kurtosis_op_network_params_template.yaml

```
@@ -0,0 +1,18 @@
+ethereum_package:
+  participants:
+    - el_type: reth
+      cl_type: lighthouse
+optimism_package:
+  chains:
+    - participants:
+      - el_type: $KURTOSIS_EL_TYPE
+        el_image: $KURTOSIS_EL_IMAGE
+        cl_type: $KURTOSIS_CL_TYPE
+        cl_image: $KURTOSIS_CL_IMAGE
+      - el_type: op-reth
+        el_image: "ghcr.io/paradigmxyz/op-reth:kurtosis-ci"
```

```suggestion
        el_image: "ghcr.io/ethereum-optimsim/op-reth:kurtosis-ci"
```
no? this should be able to run op-reth image built from branch specified in workflow dispatch

---

Yessss. I added the two dependencies as submodules to avoid having to fiddle with github permissions. When you dispatch and leave the two parameters to their defaults (`HEAD`), it will use the submodules as they are. If you add a git revision (commit hash or branch should work), it will checkout that version of the submodule.